user/models.py

```diff
@@ -0,0 +1,21 @@
+from django.db import models
+
+class User(models.Model):
+    name         = models.CharField(max_length=45)
+    email        = models.EmailField(max_length=300)
+    password     = models.CharField(max_length=300)
+    phone_number = models.CharField(max_length=45)
+    image_url    = models.URLField()
+    is_active    = models.BooleanField(default=False)
+
+    class Meta:
+        db_table = 'users'
+
+class Creator(models.Model):
+    user         = models.ForeignKey(User,on_delete=models.CASCADE)
+    image_url    = models.CharField(max_length=300)
```

Comment:
```suggestion
    image_url    = models.URLField(max_length=2000)
```
url을 저장할때에는 2000정도로 길게 주시는게 좋습니다.
장고 형식에서 지원되는  URLField 로 변경해주세요.

Reply:
수정했습니다!